Doc/whatsnew/3.11.rst

```diff
@@ -1128,33 +1128,52 @@ fcntl
 Optimizations
 =============
 
-* Compiler now optimizes simple C-style formatting with literal format
-  containing only format codes ``%s``, ``%r`` and ``%a`` and makes it as
-  fast as corresponding f-string expression.
+This section covers specific optimizations independent of the
+:ref:`whatsnew311-faster-cpython` project, which is covered in its own section.
+
+* The compiler now optimizes simple
+  :ref:`printf-style % formatting <old-string-formatting>` on string literals
+  containing only the format codes ``%s``, ``%r`` and ``%a`` and makes it as
+  fast as a corresponding :term:`f-string` expression.
   (Contributed by Serhiy Storchaka in :issue:`28307`.)
 
-* "Zero-cost" exceptions are implemented. The cost of ``try`` statements is
-  almost eliminated when no exception is raised.
-  (Contributed by Mark Shannon in :issue:`40222`.)
+* Integer division (``//``) is better tuned for optimization by compilers.
+  It is now around 20% faster on x86-64 when dividing an :class:`int`
+  by a value smaller than ``2**30``.
+  (Contributed by Gregory P. Smith and Tim Peters in :gh:`90564`.)
 
-* Pure ASCII strings are now normalized in constant time by :func:`unicodedata.normalize`.
-  (Contributed by Dong-hee Na in :issue:`44987`.)
+* :func:`sum` is now nearly 30% faster for integers smaller than ``2**30``.
```

Comment:
`sum()` doesn't appear to much faster, and still much slower than 2.7.
Maybe we shouldn't mention this, until it is fixed in 3.12

Reply:
Sure, though it still a fairly substantial improvement over 3.10, the previous feature version, which seems to me is what this section focuses on. A lot of things were slower in 3 than 2.7 (IIRC, ≈3.7 was the first version to ≈match 2.7's speed in overall benchmarks), but I'm not sure there was a general practice to only mention optimizations here if they matched or exceeded 2.7's performance.

However, I'm certainly not the expert here, and will of course defer to the judgement of @pablogsal as RM and you and rest of the core dev performance folks on whether this should be included, if you think its best left out.

Reply:
On discuss - release, Pable said "I don’t think there is anything wrong mentioning that is faster than the version in 3.10. After all there was work here to make the situation better and if true, I don’t see why a considerable improvement over the previous version should not be mentioned".  Sum is not new, but since we have an optimization section, I agree.